src/fsharp/TastOps.fs

```diff
@@ -202,6 +225,21 @@ and remapMeasureAux tyenv unt =
           else unt
        | Some (TType_measure unt) -> remapMeasureAux tyenv unt
        | Some ty -> failwithf "incorrect kinds: %A" ty
+
+and remapTupInfoAux _tyenv unt =
+    match unt with
+    | TupInfo.Const _ -> unt
+    //| TupInfo.Var tp as unt -> 
```

Comment:
I'll remove it.  It was related to [this possible extension](https://github.com/fsharp/FSharpLangDesign/blob/master/RFCs/FS-1006-struct-tuples.md#possible-extension-structness-inference)